src/SayMore/Model/Fields/FieldInstance.cs

```diff
@@ -204,11 +204,43 @@ public static IEnumerable<string> GetMultipleValuesFromText(string text)
 			var list = text.Split(new[] { kDefaultMultiValueDelimiter, kAlternateMultiValueDelimiter },
 				StringSplitOptions.RemoveEmptyEntries);
 
+			list = GetAutoCompleteList(list);
+
 			return (from val in list
 					where val.Trim() != string.Empty
 					select val.Trim());
 		}
 
+		/// ------------------------------------------------------------------------------------
+		/// <summary>
+		/// People name will be comes with role because of new feature,
+		/// We cropping the name from the list
```

Comment:
I think the comment here could be improved. Something like: "Returns names of people dropping the role from the string so that it can be used in AutoComplete"